packages/@webex/plugin-meetings/test/unit/spec/meeting/index.js

```diff
@@ -787,6 +787,36 @@ describe('plugin-meetings', () => {
           assert.calledOnce(meeting.transcription.closeSocket);
         });
       });
+      describe('#isReactionsSupported', () => {
+        it('should return false if the feature is not supported for the meeting', () => {
+          meeting.locusInfo.controls = {reactions: false};
```

Comment:
technically the correct object is `{reactions: {enabled: false }}`

Reply:
It has been working because we had in meetings/index.ts 
```
  isReactionsSupported() {
    if (this.locusInfo?.controls?.reactions) {
      return true;
    }
```

I changed it already to say
```
  isReactionsSupported() {
    if (this.locusInfo?.controls?.reactions.enabled) {
      return true;
    }
```